src/MMLib.SwaggerForOcelot/Configuration/SwaggerFileReRoute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Ocelot.Configuration.File;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MMLib.SwaggerForOcelot.Configuration
+{
+    public class SwaggerFileReRoute : FileReRoute
```

Comment:
- [ ] Missing xml comments for all public types and members.
- [ ] Rename to `SwaggerFileRoute`